main.go

```diff
@@ -209,6 +211,17 @@ func initLog(ctx *cli.Context) {
 	}
 }
 
+func initGlobalParam(ctx *cli.Context) {
+	enableWasmJitVerify := ctx.GlobalBool(utils.GetFlagName(utils.WasmVerifyMethodFlag))
```

Comment:
there are already other global params already I think, move this logic to there?